pkg/controller/machine/machine_controller.go

```diff
@@ -118,7 +118,7 @@ func (r *ReconcileMachine) Reconcile(request reconcile.Request) (reconcile.Resul
 	if errors.Cause(err) == util.ErrNoCluster {
 		klog.Infof("Machine %q in namespace %q doesn't specify %q label, assuming nil cluster", m.Name, m.Namespace, clusterv1.MachineClusterLabelName)
 	} else if err != nil {
-		return reconcile.Result{}, err
+		return reconcile.Result{}, errors.Wrapf(err, "Error getting cluster %q for machine %q in namespace %q", m.Labels[clusterv1.MachineClusterLabelName], m.Name, m.Namespace)
```

Comment:
Our convention for error messages is that they start with a lowercase letter - would you please change Error to error? Thanks!

Reply:
ok :)